extensions/amazon-aws/src/console.tsx

```diff
@@ -4,7 +4,6 @@ import { readFile } from "fs/promises";
 import { AwsAction } from "./components/common/action";
 import AWSProfileDropdown from "./components/searchbar/aws-profile-dropdown";
 import { AWS_URL_BASE } from "./constants";
-import { normalizeUrl } from "./util";
```

Comment:
Nit (non-blocking) Can you please clean up the normalizeUrl implementation if it is not being used anywhere?